freqtrade/resolvers/hyperopt_resolver.py

```diff
@@ -38,11 +38,11 @@ def __init__(self, config: Dict) -> None:
         IHyperOpt.ticker_interval = str(config['ticker_interval'])
 
         if not hasattr(self.hyperopt, 'populate_buy_trend'):
-            logger.warning("Custom Hyperopt does not provide populate_buy_trend. "
-                           "Using populate_buy_trend from DefaultStrategy.")
+            logger.warning("Hyperopt class does not provide populate_buy_trend() method. "
```

Comment:
Should we keep the "Custom" here to make it clear it's not `hyperopt.py` that causes this, but the class the user provided? 

Hyperopt is 3 things really - a Class (`Hyperopt` / `hyperopt.py`), a process (hyperopting) and a "Custom Hyperopt" (happy to hear better names)
I think this is confusing and we should change naming for this in general (oddly enough it's clear for the Strategy). Not for this PR - but was noticed here...

Reply:
actually, I wanted to ask you for removal 'custom' from the name of the cli option...
not for this pr, though.

Here the 'Custom' is not correct since it can be DefaultHyperopt, not Custom.

Reply:
well noone should use defaultHyperopt anyway, so we maybe should enforce the --customhyperopt anyway ...

Reply:
> well noone should use defaultHyperopt anyway, so we maybe should enforce the --customhyperopt anyway ...

Badabum! Really? I thought I need a special long speech to convince you in the same... 😆